Makefile

```diff
@@ -3,17 +3,20 @@
 PROJECT = probfit
 CYTHON ?= cython
 
+PYX_FILES := $(wildcard probfit/*.pyx)
+
+.PHONY: help clean build test coverage doc doc-show code-analysis flake8 pylint
+
 help:
 	@echo ''
 	@echo ' probfit available make targets:'
 	@echo ''
 	@echo '     help             Print this help message (the default)'
 	@echo ''
 	@echo '     clean            Remove generated files'
-	@echo '     build            Build inplace'
+	@echo '     build            Build Cython files'
```

Comment:
I don't care strongly, but my preference would be that

A) things are done the same in iminuit and probfit:
https://github.com/iminuit/iminuit/blob/master/Makefile#L13

B) `make build` corresponds to `python setup.py build_ext --inplace`, or we avoid the term `build` in the make targets completely to avoid confusion with people using both `setup.py` and `make`, if it's not the same.